src/overlays/actors/ovl_En_Niw_Lady/z_en_niw_lady.c

```diff
@@ -20,43 +36,535 @@ const ActorInit En_Niw_Lady_InitVars = {
+    this->unk_254 = this->unk_struct.unk_08;
+    this->unk_25A = this->unk_struct.unk_0E;
+    if (this->unk_276 == 0) {
+        Math_SmoothScaleMaxMinS(&this->unk_254.y, 0, 5, 3000, 0);
+    }
+    gSegments[6] = VIRTUAL_TO_PHYSICAL(globalCtx->objectCtx.status[this->ObjectOsAnimeIndex].segment);
+    if (this->ObjectOsAnimeIndex >= 0) {
+        if (this->unk_27E != 0) {
+            if (this->unk_26E != 0) {
+                this->unk_26E--;
+                EnNiwLady_ChoseAnimation(this, globalCtx, this->unk_26E);
+                this->unk_26E = 0;
+            }
+            SkelAnime_FrameUpdateMatrix(&this->skelAnime);
+        }
+        this->ObjectAneIndex = Object_GetIndex(&globalCtx->objectCtx, OBJECT_ANE);
+        if (this->ObjectAneIndex >= 0) {
+            this->actionFunc(this, globalCtx);
+            DECR(this->unkDownTimer1);
+            DECR(this->unkRandomDownTimer);
+            this->unkUpTimer++;
+            if (this->unkRandomDownTimer == 0) {
+                this->faceState++;
+                if (this->faceState >= 3) {
+                    this->faceState = 0;
+                    this->unkRandomDownTimer = ((s16)Math_Rand_ZeroFloat(60.0f) + 0x14);
+                }
+            }
+            func_8002E4B4(globalCtx, thisx, 20.0f, 20.0f, 60.0f, 0x1D);
+            Collider_CylinderUpdate(thisx, &this->collider);
+            if (1) {}
+            CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->collider);
+        }
+    }
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Niw_Lady/func_80ABAD38.s")
+Gfx* func_80ABB0A0(GraphicsContext* gfxCtx) {
+    Gfx* dList;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Niw_Lady/func_80ABAD7C.s")
+    dList = Graph_Alloc(gfxCtx, sizeof(Gfx));
+    gSPEndDisplayList(dList);
+    return dList;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Niw_Lady/EnNiwLady_Update.s")
+s32 EnNiwLady_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot,
+                               Actor* thisx) {
+    EnNiwLady* this = THIS;
+    s32 pad;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Niw_Lady/func_80ABB0A0.s")
+    if (limbIndex == 15) {
+        rot->x += this->unk_254.y;
+        rot->z += this->unk_254.x;
+    }
+    if (limbIndex == 8) {
+        rot->x += this->unk_25A.y;
+    }
+    if (this->unk_275 != 0) {
+        if ((limbIndex == 8) || (limbIndex == 10) || (limbIndex == 13)) {
+            rot->y += (Math_Sins((globalCtx->state.frames * ((limbIndex * 0x32) + 0x814))) * 200.0f);
+            rot->z += (Math_Coss((globalCtx->state.frames * ((limbIndex * 0x32) + 0x940))) * 200.0f);
+        }
+    }
+    return 0;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Niw_Lady/func_80ABB0CC.s")
+void EnNiwLady_Draw(Actor* thisx, GlobalContext* globalCtx) {
+    static Gfx* D_80ABB408[] = { 0x060008C8, 0x060010C8, 0x060018C8 };
```

Comment:
Any reason these display lists aren't external symbols? I'm not familiar with the OOT standards (yet), but in MM I like to see symbols created.

Reply:
not sure if we talked about this yet, but in oot we only use undefined syms for the purposes of matching. When the object is decompiled it will have its own symbol created separately, and replaced properly. So there isnt much accomplished by making this a symbol in the short term.

I know mm does it differently, but yea